vip_hci/hci_dataset.py

```diff
@@ -45,6 +48,12 @@ class HCIFrame:
         The FWHM associated with this dataset (instrument dependent). Required
         for several methods (operations on the cube).
     """
+
+
+    SAVE_WITH_NP = ["image"]
```

Comment:
Why is this attribute name in capital letters? Could it we include in `__init__` or in the method signature where it's used?

Reply:
I suggest to leave it as it is for now — I created a new `Saveable` super-class which wraps all that redundant `save` related things, and adds support for keras models etc. We could discuss that on the future PR, when the `Saveable` class is ready?